test/unit/sync.coffee

```diff
@@ -217,6 +217,8 @@ describe "Sync", ->
 
 
     describe 'applied', ->
+        @timeout 5000
```

Comment:
I thought it was 5s, not 10s in this file. And I think we should modify it to come back to 5s. 5s is already much more than we need. And I breaks the flow to have to wait 10s to see a test fails in local when you forgot a callback. For travis, it won't change anything. When the 5s timeout is reached, it's nearly always because something has broken in a previous test.